Sources/PackageDescription4/Package.swift

```diff
@@ -216,6 +240,18 @@ public final class Package {
         registerExitHandler()
     }
   #else
+    /// Initializes and returns a newly allocated package object with the provided configuration options.
+    ///
+    /// - Parameters:
+    ///     - name: The name of the Swift package.
+    ///     - pkgConfig: Additional flags for a system package.
+    ///     - providers: The package providers for a system package.
```

Comment:
Let's remove doc comments for pkgConfig and providers from initializer for 4.2 and 5 since they are deprecated.

Reply:
I'm not sure I understand this correctly, do you want me to remove the whole docs comment (lines 244-255) or just for them to be

```
    /// Initializes and returns a newly allocated package object with the provided configuration options.
    ///
    /// - Parameters:
    ///     - name: The name of the Swift package.
    ///     - products: The list of products that this package vends and that can be run or used by its clients.
    ///     - dependencies: The list of package dependencies.
    ///     - targets: The list of targets that are part of this package.
    ///     - swiftLanguageVersions: The list of Swift versions that this package is compatible with.
    ///     - cLanguageStandard: The C language standard to use for all C targets in this package.
    ///     - cxxLanguageStandard: The C++ language standard to use for all C++ targets in this package.```?

Reply:
Just them be:

```
 /// Initializes and returns a newly allocated package object with the provided configuration options.
    ///
    /// - Parameters:
    ///     - name: The name of the Swift package.
    ///     - products: The list of products that this package vends and that can be run or used by its clients.
    ///     - dependencies: The list of package dependencies.
    ///     - targets: The list of targets that are part of this package.
    ///     - swiftLanguageVersions: The list of Swift versions that this package is compatible with.
    ///     - cLanguageStandard: The C language standard to use for all C targets in this package.
    ///     - cxxLanguageStandard: The C++ language standard to use for all C++ targets in this package.
```

Reply:
I removed them.